apollo-gradle-plugin/src/test/files/gradle/build.gradle.kts.template

```diff
@@ -18,8 +18,8 @@ buildscript {
     maven {
       url = uri("../../../build/localMaven")
     }
-    google()
     mavenCentral()
+    google()
```

Comment:
I moved `google()` down as part of this PR because the Google maven repo was having issues and it shouldn't hurt but this is not related to `apollo-gradle-plugin-external`